lib/sycamore/sycamore/transforms/sort.py

```diff
@@ -6,6 +6,31 @@
 if TYPE_CHECKING:
     from ray.data import Dataset
 
+from sycamore.plan_nodes import UnaryNode
+
+
+class DropIfMissingField(UnaryNode):
```

Comment:
Can this not just be a filter function?

Reply:
Nope, it needs to remove metadata documents also and there isn't an easy way to get at metadata as it flows through (by design you're not supposed to do what I'm doing)